src/modules/rover_pos_control/RoverPositionControl.cpp

```diff
@@ -265,6 +265,37 @@ RoverPositionControl::control_position(const matrix::Vector2f &current_position,
 	return setpoint;
 }
 
+bool
+RoverPositionControl::control_velocity(const matrix::Vector2f &current_velocity,
+				       const matrix::Vector3f &ground_speed, const position_setpoint_triplet_s &pos_sp_triplet)
+{
+	bool setpoint = true;
+	float mission_throttle = _param_throttle_cruise.get();
```

Comment:
Can be `const`. And many others below can be as well.

Reply:
Makes sense thanks!